pom.xml

```diff
@@ -103,8 +100,13 @@
 			<version>${json-patch.version}</version>
 		</dependency>
 		<dependency>
-			<groupId>org.keycloak</groupId>
-			<artifactId>keycloak-spring-boot-starter</artifactId>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-oauth2-resource-server</artifactId>
+		</dependency>
+		<dependency>
```

Comment:
Is this needed? If so can we add the version to the properties?

Reply:
1) We need it for the JwtAuthConverter
2) Version is managed by spring boot, it's at 3.1.0 right now